tests/components/shelly/test_update.py

```diff
@@ -61,14 +61,16 @@ async def test_block_update(
     monkeypatch.setitem(mock_block_device.status, "cloud", {"connected": False})
     await init_integration(hass, 1)
 
-    state = hass.states.get(entity_id)
+    assert (state := hass.states.get(entity_id))
     assert state.state == STATE_ON
     assert state.attributes[ATTR_INSTALLED_VERSION] == "1.0.0"
     assert state.attributes[ATTR_LATEST_VERSION] == "2.0.0"
     assert state.attributes[ATTR_IN_PROGRESS] is False
     assert state.attributes[ATTR_UPDATE_PERCENTAGE] is None
-    supported_feat = state.attributes[ATTR_SUPPORTED_FEATURES]
```

Comment:
I don't understand why this change it needed, it just make the code less readable

Reply:
In so many places we forced to not create a variable for a single usage.